crates/pep508-rs/src/marker/parse.rs

```diff
@@ -256,8 +256,10 @@ pub(crate) fn parse_marker_key_op_value<T: Pep508Url>(
 /// relatively rare so the trade-off is acceptable.
 ///
 /// The following limited expression is supported:
-///     
+///
+/// ```text
 ///      [not] in '<version> [additional versions]'
```

Comment:
Maybe doesn't need to be indented then?

Reply:
Aye. And I caught another one in `uv-resolver`.